packages/share-transfer/src/ShareTransferModule.ts

```diff
@@ -51,7 +51,7 @@ class ShareTransferModule implements IModule {
 
     // This is needed to avoid MIM during share deletion.
     if (numberOfNewShares <= numberOfOldShares) {
-      const shareTransferStorePointer: ShareTransferStorePointer = { pointer: new BN(generatePrivate()) };
+      const shareTransferStorePointer: ShareTransferStorePointer = { pointer: new BN(generatePrivate(KeyType.secp256k1)) };
```

Comment:
I assume `secp256k1` is intended here. Why does this functionality not depend on key type? Is this just used for encryption? Maybe create a function `generateEncryptionKey`?

Reply:
Hmm... this may be a bug here. Will re-check it.

Reply:
You are correct that secp256k1 is intended here though.